### ISSUE TITLE
Order APi

### ISSUE THREAD

**Issue by yyyakupyazici** [allegro/allegro-api#10525]
When I query order with api ('https://api.allegro.pl/order/checkout-forms')  It returns nothing. But I can display my orders on my allegro account. My traceId is 3750a3b79d3b45a6. Can you check please ?

**Comment by PrzemyslawLukanowski** [comment]
You are authorized as user 120****55 - and this account has no orders. I assume you must be logged in as another user on the website. You can send me a sample order ID from the website, I will check it out.

**Comment by PrzemyslawLukanowski** [comment]
Yes, this order belongs to a different seller.

**Comment by PrzemyslawLukanowski** [comment]
It's the same situation, you are authorized as a user 120****55 and the order you mentioned belongs to another merchant.

**Comment by yyyakupyazici** [comment]
sorry for mistake. My traceId is 2f48e597cfea033e and my orderId is 58561030-bd85-11ef-802a-63766fea07ed Could you check again please?

**Comment by PrzemyslawLukanowski** [comment]
Still the same situation, the order belongs to a different seller.

**Comment by yyyakupyazici** [comment]
I have attached a screenshot of the order displayed on my page. The trace ID associated with the order query response is 5600e95b86047f30. Could you please verify if the issue persists and provide further assistance?




![Screenshot 2024-12-20 at 00 09 03](https://github.com/user-attachments/assets/96e5f4d7-3d3e-46b9-90d5-984038abd55f)

**Comment by PrzemyslawLukanowski** [comment]
This order belongs to seller 1149xxx16, and you are using a token of a different user - 1200xxx55. You need to authorize the application as a user 1149xxx16, thanks to that you will generate tokens for this specific user and you will see this order. Now you perform operations as user 1200xx55, who doesn't have any orders.

**Comment by yyyakupyazici** [comment]
I get token with clientId below. This clientId is butikzofia's. I created yesterday. 

clientId: d30b03fb58c647afb359eb6bb1ef8024

**Comment by PrzemyslawLukanowski** [comment]
Yes, but when you were authorizing an application using https://allegro.pl/auth/oauth/authorize... you were logged in as a different seller. One application can be created by a specific account, but it can be used by multiple different accounts, not exactly by the account that created a client ID - this happened it this situation. You created the clientID with account "A", but the account "B" authorized it and gave permission to operate on his behalf.

**Comment by yyyakupyazici** [comment]
Ok. So why order api return empty order list? This account may have order. Can you share any orderId that gotten with this clientId?

**Comment by PrzemyslawLukanowski** [comment]
As I mentioned - API returns an empty array because you are authorized as the user who doesn't have any orders. You are authorized as a user 1200xxx55 who is not an owner of the clientId d30b03fb58c647afb359eb6bb1ef8024. Please look at the thread #10555 -this is exactly the same case.

**Comment by yyyakupyazici** [comment]
Thanks. The problem is resolved.
